pkgs/applications/misc/mdp/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchgit, ncurses }:
+
+stdenv.mkDerivation {
+  name = "mdp-0.93";
+
+  src = fetchgit {
```

Comment:
I was once told to use `fetchFromGitHub` instead. Take that for what it's worth.